### PR TITLE
Add dynamic move threshold to pnz touch detection

### PR DESCRIPTION
**Pull request by zbettenbuk**
This PR makes the move threshold in the PnZ feature more dynamic

### CODE REVIEW

File: react/features/base/media/components/native/VideoTransform.js [hunk 1]
```diff
@@ -167,6 +173,13 @@ class VideoTransform extends Component<Props, State> {
      * @inheritdoc
      */
     componentWillMount() {
+        const { height, width } = Dimensions.get('window');
```
Comment (saghul) [inline react/features/base/media/components/native/VideoTransform.js]: Note that the window dimensions might not be the app dimensions. See https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/responsive-ui/components/DimensionsDetector.native.js for dynamically detecting dimensions.

Comment (zbettenbuk) [reply react/features/base/media/components/native/VideoTransform.js]: In which cases can window dimensions be different from the app screen size?

Comment (paweldomas) [reply react/features/base/media/components/native/VideoTransform.js]: Probably if you double press home button on iOS which bring the app half screen. I don't know that for sure, but there are also some split window modes on tablets or something like that.

Comment (saghul) [reply react/features/base/media/components/native/VideoTransform.js]: split view, for insatance

Comment (zbettenbuk) [reply react/features/base/media/components/native/VideoTransform.js]: Ok, added a more flexible solution :)

Comment (paweldomas) [reply react/features/base/media/components/native/VideoTransform.js]: So this gets the dimensions on componentDidMount, but resize will probably not unmount component and the value will be outdated. The docs[1] advise to call it on every render or maybe this DimensionDetector mentioned by @saghul will work, but you need to check how it works as I don't know that...
[1]: https://facebook.github.io/react-native/docs/dimensions.html#get

File: react/features/base/media/components/native/VideoTransform.js [hunk 2]
```diff
@@ -167,6 +173,12 @@ class VideoTransform extends Component<Props, State> {
      * @inheritdoc
      */
     componentWillMount() {
+        // The move threshold should be adaptive to the pixel ratio of the
+        // screen to avoid making it too sensitive or difficult to handle on
+        // different pixel ratio screens.
+        this.moveThreshold
+            = PixelRatio.get() * MOVE_THRESHOLD_DISMISSES_TOUCH;
```
Comment (saghul) [inline react/features/base/media/components/native/VideoTransform.js]: This could have been a static constant, since it cannot ever change, right?

**Pull request merged**